.vale/fixtures/RedHat/Spelling/testvalid.adoc

```diff
@@ -128,6 +132,7 @@ I/O
 IDE
 IDEs
 Infinispan
+Ignition config
```

Comment:
Because "Ignition config" is always capitalized this way, perhaps we should also test for proper capitalization.

Reply:
Added an entry to the `CaseSensitiveTerms.yml` list